test/e2e/functional_test.go

```diff
@@ -945,7 +945,7 @@ func (s *FunctionalSuite) TestPauseBefore() {
 		Workflow(`@functional/pause-before.yaml`).
 		When().
 		SubmitWorkflow().
-		WaitForWorkflow(fixtures.ToBeRunning).
```

Comment:
why changed?

Reply:
This test was flaky. It uses `kubectl exec` which won't work unless the pod is running.